gotocompany/optimus/core/v1beta1/job_spec.proto

```diff
@@ -518,3 +534,36 @@ message GetWindowResponse {
   google.protobuf.Timestamp start = 1;
   google.protobuf.Timestamp end = 2;
 }
+
+enum JobState {
+  JOB_STATE_UNSPECIFIED = 0;
+  JOB_STATE_ENABLED = 1;
+  JOB_STATE_DISABLED = 2;
+}
+
+message UpdateJobsStateRequest {
+  string project_name = 1;
+  string namespace_name = 2;
+  string remark = 3;
+  JobState state = 4;
+  repeated string job_names = 5;
+}
+
+message UpdateJobsStateResponse {
+
+}
+
+message SyncJobsStateRequest {
+  string project_name = 1;
+  string namespace_name = 2;
+  message JobStates {
```

Comment:
`JobStates`looks like it has multiple `JobState` while it is not. should we rename it to `JobStatePair` or `JobStateItem` or something else?

apart from that, changes looks good